index.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="style.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" 
+integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" 
+crossorigin="anonymous" referrerpolicy="no-referrer">
+<title>73rd Fifa congress</title>
+</head>
+<body>
+  <header class="header-header">
+    <ul class="header-list">
+      <li>
+        <a href="#">
+          <i class="fa-brands fa-facebook"></i>
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <i class="fa-brands fa-twitter"></i>
+        </a>
+      </li>
+      <li class="header-link"><a href="#Mypage">My Page</a></li>
+      <li class="header-link"><a href="#">English</a></li>
+    </ul> 
+  </header>
+  <nav class="nav">
+    <i class="fa-solid fa-bars menu-menu" id="menu-icon"></i>
+    <img src="./images/page-images/FIFA.png" alt="73rd Congress logo" class="img-logo">
+    <div class="big-screen-list" id="nav-menu">  
+      <ul class="nav-desk-list">
+        <li class="bg-list-item">
+          <a href="about.html">About</a>
+        </li>
+        <li class="bg-list-item">
+          <a href="index.html">Home</a>
+        </li>
+        <li class="bg-list-item">
+          <a href="#Program">Program</a>
+        </li>
+        <li class="bg-list-item">
+          <a href="#partner">Partners</a>
+        </li>
+        <li class="bg-list-item extra">
+          <a href="https://www.fifa.com/social-impact/campaigns">FIFA Campain</a>
+        </li>
+      </ul>
+      <i class="fa-solid fa-xmark close-button" id="close-icon"></i>
+    </div>
+  </nav>
+  <main>
+    <section class="first-class">
+      <div class="first-container">
+        <h5 class="sub-header">"Hello! Sharing world"</h5>
+        <h1 class="main-title">73RD FIFA CONGRESS MEETING</h1>
```

Comment:
- [ ] Kindly ensure that you use the cont sizes and colors for this section to match the ones that are provided in te design guidelines.

**This is how your design looks like:**

![image](https://user-images.githubusercontent.com/86318284/229808731-be7f445e-66e2-4c9a-bd8c-cd2b5453696c.png)


**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/229808643-1ad99a68-3e9d-4165-9fea-980a04bf4eff.png)